api/preprints/views.py

```diff
@@ -108,20 +97,26 @@ class PreprintList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin):
         Query Params:  <none>
         Body (JSON):   {
                         "data": {
-                            "id": node_id,
-                            "attributes": {
-                                "subjects":      [{subject_id}, ...]  # required
-                                "description":   {description},       # optional
-                                "tags":          [{tag1}, ...],       # optional
-                                "provider":      {provider}           # optional
-                            },
+                            "attributes": {},
                             "relationships": {
-                                "primary_file": {                     # required
+                                "node": {                           # required
                                     "data": {
-                                        "type": "primary",
+                                        "type": "node",
```

Comment:
Good catch. Updated in https://github.com/CenterForOpenScience/osf.io/pull/6346/commits/5e6feee9b0c1fb6a8457ff4502605ad74bd2d9a2